Update tor.md

sambradbury opened the pull request:
Typo correction and further clarification for mac users

sambradbury committed:
Update tor.md

Typo correction and further clarification for mac users

benthecarman commented on docs/tor/tor.md:
```diff
@@ -88,13 +88,14 @@ config, as well as have tor configured for it.
 ControlPort 9051
 CookieAuthentication 1
 CookieAuthFileGroupReadable 1
+CookieAuthFile /usr/local/var/tor/control_auth_cookie
```
we should probably move these 2 setting to an optional setting section as they aren't always needed. Also the path should be different for each os

sambradbury committed:
Update tor.md

Moved torrc CooieAuthFile setting to the newly created 'optional settings section' and added extra clarity for the issue described here (https://github.com/bitcoin-s/bitcoin-s/issues/3547) which is similar to the issue I experienced.
